### ISSUE TITLE
Please add al the files in HTML

### ISSUE THREAD

**Issue by arnav2811** [whitehatjr/p5.play-boilerplate#10]
it would be easier for all the students if you add the files in html or at least inform us that it is not included

**Comment by SpyGuy0215** [comment]
yes, please include it as it is tedious to do so when it can be in here already
